packages/cxl-lumo-styles/scss/colors.scss

```diff
@@ -0,0 +1,94 @@
+  --lumo-tint-30pct: hsla(0, 0%, 100%, 0.5);
+  --lumo-tint-40pct: hsla(0, 0%, 100%, 0.57);
+  --lumo-tint-50pct: hsla(0, 0%, 100%, 0.64);
+  --lumo-tint-60pct: hsla(0, 0%, 100%, 0.7);
+  --lumo-tint-70pct: hsla(0, 0%, 100%, 0.77);
+  --lumo-tint-80pct: hsla(0, 0%, 100%, 0.84);
+  --lumo-tint-90pct: hsla(0, 0%, 100%, 0.9);
+  --lumo-tint: #fff;
+  /* Shade */
+  --lumo-shade-5pct: hsla(214, 61%, 25%, 0.05);
+  --lumo-shade-10pct: hsla(214, 57%, 24%, 0.1);
+  --lumo-shade-20pct: hsla(214, 53%, 23%, 0.16);
+  --lumo-shade-30pct: hsla(214, 50%, 22%, 0.26);
+  --lumo-shade-40pct: hsla(214, 47%, 21%, 0.38);
+  --lumo-shade-50pct: hsla(214, 45%, 20%, 0.5);
+  --lumo-shade-60pct: hsla(214, 43%, 19%, 0.61);
+  --lumo-shade-70pct: hsla(214, 42%, 18%, 0.72);
+  --lumo-shade-80pct: hsla(214, 41%, 17%, 0.83);
+  --lumo-shade-90pct: hsla(214, 40%, 16%, 0.94);
+  --lumo-shade: hsl(214, 35%, 15%);
+  /* Contrast */
+  --lumo-contrast-5pct: var(--lumo-shade-5pct);
+  --lumo-contrast-10pct: var(--lumo-shade-10pct);
+  --lumo-contrast-20pct: var(--lumo-shade-20pct);
+  --lumo-contrast-30pct: var(--lumo-shade-30pct);
+  --lumo-contrast-40pct: var(--lumo-shade-40pct);
+  --lumo-contrast-50pct: var(--lumo-shade-50pct);
+  --lumo-contrast-60pct: var(--lumo-shade-60pct);
+  --lumo-contrast-70pct: var(--lumo-shade-70pct);
+  --lumo-contrast-80pct: var(--lumo-shade-80pct);
+  --lumo-contrast-90pct: var(--lumo-shade-90pct);
+  --lumo-contrast: var(--lumo-shade);
+  /* Text */
+  --lumo-header-text-color: var(--lumo-contrast);
+  --lumo-body-text-color: var(--lumo-contrast-90pct);
+  --lumo-secondary-text-color: var(--lumo-contrast-70pct);
+  --lumo-tertiary-text-color: var(--lumo-contrast-50pct);
+  --lumo-disabled-text-color: var(--lumo-contrast-30pct);
+  /* Primary */
+  --lumo-primary-color: hsl(8, 100%, 39%);
+  --lumo-primary-color-50pct: hsla(8, 100%, 39%, 0.5);
+  --lumo-primary-color-10pct: hsla(8, 100%, 39%, 0.1);
+  --lumo-primary-text-color: var(--lumo-primary-color);
+  --lumo-primary-contrast-color: #fff;
+  /* Error */
+  --lumo-error-color: hsl(8, 100%, 39%);
+  --lumo-error-color-50pct: hsla(8, 100%, 39%, 0.5);
+  --lumo-error-color-10pct: hsla(8, 100%, 39%, 0.1);
+  --lumo-error-text-color: hsl(8, 100%, 39%);
+  --lumo-error-contrast-color: #fff;
+  /* Success */
+  --lumo-success-color: hsl(8, 100%, 39%);
+  /* hsl(144,82%,37%); */
+  --lumo-success-color-50pct: hsla(8, 100%, 39%, 0.55);
+  --lumo-success-color-10pct: hsla(8, 100%, 39%, 0.12);
+  --lumo-success-text-color: hsl(8, 100%, 39%);
+  --lumo-success-contrast-color: #fff;
+}
+
+html {
+  color: var(--lumo-body-text-color);
+  background-color: var(--lumo-base-color);
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  color: var(--lumo-header-text-color);
+}
+
+a {
+  color: var(--lumo-primary-text-color);
+}
+
+blockquote {
+  color: var(--lumo-secondary-text-color);
+}
+
+code,
+pre {
+  background-color: var(--lumo-contrast-10pct);
+  border-radius: var(--lumo-border-radius-m);
+}
```

Comment:
Are these really all our overrides?

Reply:
what you can see in colors.scss is override

Reply:
Jaan delivered this override set, or? I'm not sure I've ever seen so many overrides before.